src/wallet/migrate.cpp

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2024 The Bitcoin Core developers
```

Comment:
I see you've removed the `2024-present` from the latest push - is that a general direction from now on?